programs/driver_board/main.c

```diff
@@ -7,11 +7,17 @@
 
 #include "communication.h"
 
+#define MOTOR_PWM_CHANNEL 3
+
 #define UNUSED(arg) (void)(arg)
 
 thread_t *commanderp;
 
+int16_t motor_speed = 0;
+
 void execute_master_command(uint16_t command_id, uint8_t *buff) {
+    int16_t motor_speed;
```

Comment:
Note that this local variable shadows the global one, making any changes to `motor_speed` within this function pointless if we want to work with them anywhere else (for example in `MotorThread`).